test/DebugInfo/modulecache.swift

```diff
@@ -15,7 +15,7 @@ import ClangModule
 
 // RUN: %empty-directory(%t)
 // RUN: %target-swift-frontend %s -c -g -o %t.o -module-cache-path %t -I %S/Inputs
-// RUN: file %t/*/ClangModule-*.pcm | egrep -q '(Mach-O|ELF)'
+// RUN: llvm-readobj -h %t/*/ClangModule-*.pcm | egrep -q 'Format: (Mach-O|ELF|COFF)-(.*)'
```

Comment:
Seems also silly to use egrep instead of FileCheck here. Would you mind fixing that as well?

Reply:
Sure, I can do that as well.

Reply:
Well, yes, the `egrep` seems excessive, but, if you had multiple tests here, it wouldn't necessarily work if you are using different formats.  I don't feel too strongly one way or the other though.